package.json

```diff
@@ -17,6 +17,7 @@
     "storyshots": "cd packages/server-web && npm run test:storyshots",
     "upload-coverage": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
     "combine-coverage": "istanbul report --root coverage",
+    "codeceptjs": "./node_modules/.bin/codeceptjs run --steps",
```

Comment:
This should be `test:browser` or something like that?

Reply:
yes, but then it would be run together with all the other tests. this will not work, because those are run inside the docker container, where we don't have a xvfb or chrome.